trackpy/tests/test_static.py

```diff
@@ -0,0 +1,146 @@
+
+from trackpy.static import pairCorrelation2D, pairCorrelation3D
+from trackpy.static import _points_ring3D
+import unittest
+import pandas
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+class TestPairCorrelation(unittest.TestCase):
+
+
+    def test_correlation2D_lattice(self):
+        ### Lattice Test
+        # With proper edge handling, g(r) of the particle at the center should be the same as g(r) for all particles.
+        lattice = self._lattice2D()
+
+        # Calculate g_r on the center particle only (index 210)
+        edges, g_r_one = pairCorrelation2D(lattice, dr=.1, cutoff=8, p_indices=[210])
+        g_r_one /= np.linalg.norm(g_r_one) #We care about the relative difference of g_r in this case, so let's normalize both.
```

Comment:
Bikeshedding: you might import `norm` in the root namespace (`from numpy.linalg import norm`) as it is used many times.